spec/jobs/characterize_job_spec.rb

```diff
@@ -1,66 +1,34 @@
 RSpec.describe CharacterizeJob do
-  let(:file_set_id) { 'abc12345' }
-  let(:filename)    { Rails.root.join('tmp', 'uploads', 'ab', 'c1', '23', '45', 'abc12345', 'picture.png').to_s }
-  let(:file_set) do
-    FileSet.new(id: file_set_id).tap do |fs|
-      allow(fs).to receive(:original_file).and_return(file)
-      allow(fs).to receive(:update_index)
+  context 'when the work is an ActiveFedora FileSet' do
```

Comment:
Is this a context we want to keep here? the job is not switching behavior anymore.